includes/definitions/discovery/vrp.yaml

```diff
@@ -12,23 +12,31 @@ modules:
                     value: hwEntityOpticalTemperature
                     num_oid: .1.3.6.1.4.1.2011.5.25.31.1.1.3.1.5.
                     descr: '{{ $entPhysicalName }}'
-                    index: 'hwEntityOpticalTemperature.{{ $index }}'
+                    index: '{{ $index }}'
                     skip_value_lt: -50
                 -
                     oid: hwEntityStateTable
                     value: hwEntityTemperature
                     num_oid: .1.3.6.1.4.1.2011.5.25.31.1.1.1.1.11.
                     descr: '{{ $entPhysicalName }}'
-                    index: 'hwEntityTemperature.{{ $index }}'
+                    index: '{{ $index }}'
                     skip_values: 0
         dbm:
             data:
                 -
                     oid: hwOpticalModuleInfoTable
                     value: hwEntityOpticalRxPower
                     num_oid: .1.3.6.1.4.1.2011.5.25.31.1.1.3.1.8.
-                    descr: '{{ $entPhysicalName }}'
-                    index: 'hwEntityOpticalRxPower.{{ $index }}'
+                    descr: '{{ $entPhysicalName }} Rx'
+                    index: 'rx-{{ $index }}'
```

Comment:
This one is just `{{ $index }}`

Reply:
old file have `` 'rx-' . $index ``

Reply:
You're right, when I looked earlier I thought it didn't